.isort.cfg

```diff
@@ -0,0 +1,10 @@
+[settings]
+profile=black
+from_first=true
+import_heading_future=Future
+import_heading_stdlib=Standard
+import_heading_thirdparty=Third Party
+import_heading_firstparty=First Party
+import_heading_localfolder=Local
+known_firstparty=alog,aconfig,caikit,caikit_computer_vision,import_tracker
```

Comment:
NIT: This looks like `caikit_computer_vision` is listed in both `firstparty` and `localfolder`

Reply:
Good catch, thanks!